### PR TITLE
bash-completion: autocomplete cgroup names in systemd-cgtop

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: shell-completion/bash/systemd-cgtop [hunk 1]
```diff
@@ -56,6 +56,9 @@ _systemd_cgtop() {
     fi
 
     COMPREPLY=( $(compgen -W '${OPTS[*]}' -- "$cur") )
+    if [ -d /sys/fs/cgroup/ ]; then
+        COMPREPLY+=( $(cd /sys/fs/cgroup/ && compgen -o nospace -o dirnames "$cur") )
```
Comment (keszybz) [inline shell-completion/bash/systemd-cgtop:62]: This does not work with cgroup v1. Do we care?

Comment (bluca) [reply shell-completion/bash/systemd-cgtop:62]: Speaking for myself, I really don't :-) I can add a quick check to ensure it's in unified mode?

Comment (keszybz) [reply shell-completion/bash/systemd-cgtop:62]: Maybe try to cd to /sys/fs/cgroup/systemd first, and then /sys/fs/cgroup otherwise?

Comment (bluca) [reply shell-completion/bash/systemd-cgtop:62]: ok, done

**Review by DaanDeMeyer — APPROVED** [review]

**Pull request merged**